exercises/concept/boutique-inventory/.meta/exemplar.rb

```diff
@@ -4,7 +4,10 @@ def initialize(items)
   end
 
   def item_names
-    items.map { |item| item[:name] }.sort
```

Comment:
Does it make sense to get a listing of items that we do not have on hand?  It is an item name, not a verification that it is in stock, maybe?  I have to read the README to know if that is a consideration, but on the face of this change, we are changing this behavior considerably.

Reply:
The README is confusing in this aspect, since it shows "Red Short Skirt" as part of the output,  when it has none in stock, but it does say for point 1 that it should "Return a list of the names of the items in stock."

We should probably remove "in stock" and have it not exclude 0 stock items.  I vote for a change in the README to get rid of the conflicting information.  I prefer to have `item_names` give a listing of the names of items, regardless of stock level.

Reply:
To be sure, though, I have made the statement "We have zero in stock."  So even none is a stock level.  We list it as something that we do stock and having 0 in stock makes sense.